bootstrapper/internal/helm/client.go

```diff
@@ -25,39 +25,38 @@ import (
 	corev1 "k8s.io/api/core/v1"
 )
 
-type HelmClient struct {
+const (
+	// timeout is the maximum time given to the helm client.
+	timeout = 5 * time.Minute
+	// kubeConfig is the path to the Kubernetes admin config (used for authentication).
+	kubeConfig = "/etc/kubernetes/admin.conf"
+)
+
+type Client struct {
 	*action.Install
 }
 
-func NewHelmUtil(log *logger.Logger) (*HelmClient, error) {
+func NewClient(log *logger.Logger) (*Client, error) {
```

Comment:
```suggestion
func New(log *logger.Logger) (*Client, error) {
```
Since this is the only init function in this package, it should be shortened to just `New`.
Also missing a doc comment